en/components/tabbar.md

```diff
@@ -342,6 +342,154 @@ And don't forget to include the following CSS definitions in order for the appli
+}
+```
+
+The next step is to import these components in your `app.module.ts` file: 
+
+```typescript
+// app.module.ts
+import {
+    BottomNavRoutingComponent,
+    BottomNavRoutingView1Component,
+    BottomNavRoutingView2Component,
+    BottomNavRoutingView3Component } from './bottomnav-routing.component';
+import { AppRoutingModule } from "./app-routing.module";
+
+...
+
+@NgModule({
+    declarations: [
+        BottomNavRoutingComponent,
+        BottomNavRoutingView1Component,
+        BottomNavRoutingView2Component,
+        BottomNavRoutingView3Component
+    ],
+    imports: [
+        AppRoutingModule
+    ]
+})
+export class AppModule { }
+```
+
+Also do not forget to create the appropriate navigation mappings in the `app-routing.module.ts` file:
+
+```typescript
+// app-routing.module.ts
+import {
+    BottomNavRoutingComponent,
+    BottomNavRoutingView1Component,
+    BottomNavRoutingView2Component,
+    BottomNavRoutingView3Component } from './bottomnav-routing.component';
+
+...
+
+const appRoutes = [
+    {
+        path: '',
+        pathMatch: 'full',
+        redirectTo: '/bottomnav-routing'
+    },
+    {
+        path: 'bottomnav-routing',
+        component: BottomNavRoutingComponent,
+        children: [
+            { path: 'view1', component: BottomNavRoutingView1Component },
+            { path: 'view2', component: BottomNavRoutingView2Component },
+            { path: 'view3', component: BottomNavRoutingView3Component },
+        ]
+    }
+];
+
+@NgModule({
+    exports: [RouterModule],
+    imports: [RouterModule.forRoot(appRoutes)]
+})
+export class AppRoutingModule { }
+```
+
+Now that we have all navigation routes setup, we need to declare the BottomNavigation component and configure it for routing.
+Also, make sure to add a router-outlet for rendering the view components' output.
+
+```html
+<!-- bottomnav-routing.component.html -->
+<router-outlet></router-outlet>
+
+<igx-bottom-nav>
+
+    <igx-tab label="Tab 1" icon="dashboard"
+		routerLink="view1"
+		routerLinkActive #rla1="routerLinkActive"
+		[isSelected]="rla1.isActive">
+	</igx-tab>
+
+	<igx-tab label="Tab 2" icon="check_circle_outline"
+		routerLink="view2"
+		routerLinkActive #rla2="routerLinkActive"
+		[isSelected]="rla2.isActive">
+	</igx-tab>
+
+	<igx-tab label="Tab 3" icon="radio_button_checked"
+		routerLink="view3"
+		routerLinkActive #rla3="routerLinkActive"
+		[isSelected]="rla3.isActive">
+	</igx-tab>
+
+</igx-bottom-nav>
+```
+
+The above code creates a BottomNavigation component with three tab items. All tab items are having the `RouterLink` directive applied which is used to specify the routing link used for the navigation. If any of these links became active, the corresponding tab item will have its `isSelected` property set because of the binding to the `RouterLinkActive` directive's `isActive` property. This way the selected tab item will always stay synchronized with the current browser's address.
```

Comment:
"If any of these links became active" - this should be changed to present simple tense (since future simple is used afterwards) -> "If any of these links becomes active"

Reply:
Addressed.